Makefile

```diff
@@ -14,16 +14,7 @@ build-docker:
 
 build-cli: clean
 	-mkdir ./cli/build
-	cd ./cmd/migrate && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o ../../cli/build/migrate.linux-amd64 -ldflags='-X main.Version=$(VERSION) -extldflags "-static"' -tags '$(DATABASE) $(SOURCE)' .
```

Comment:
I am assuming we are planning to contribute this fix upstream and delete our fork eventually? If so, any reason to remove these build instructions for the various platforms that the upstream will be expecting to build?

Reply:
Yes the idea is to eventally contribute a fix upstream. Since we are going to be using this fork for now I removed the other builds becuase we are calling `make build-cli` from the dockerfile which really slows up our docker build so I got rid of them since this is only our fork and building for windows and macos are not needed.